lib/ts/ink_queue.cc

```diff
@@ -52,6 +52,11 @@
 #include "ts/hugepages.h"
 #include "ts/Diags.h"
 
+
+// in order to compile correctly
```

Comment:
This comment warns against unknown danger. Can you expound upon what "compile correctly" means in this context? Why must we undefine that symbol? Is there no better way?